src/components/Comment/Comment.tsx

```diff
@@ -154,35 +146,35 @@ const linkStyles = (pillar: Pillar) => css`
   }
 `;
 
+const flexGrowStyles = css`
+  flex-grow: 1;
+`;
+
+const spaceBetweenStyles = css`
+  display: flex;
+  justify-content: space-between;
+`;
+
 const flexRowStyles = css`
   display: flex;
   flex-direction: row;
 `;
 
+const flexColStyles = css`
+  display: flex;
+  flex-direction: column;
+`;
+
 const removePaddingLeft = css`
   padding-left: 0px;
 `;
 
 const Column = ({ children }: { children: JSX.Element | JSX.Element[] }) => (
-  <div
-    className={css`
-      display: flex;
-      flex-direction: column;
-    `}
-  >
-    {children}
-  </div>
+  <div className={flexColStyles}>{children}</div>
 );
 
 const Row = ({ children }: { children: JSX.Element | JSX.Element[] }) => (
-  <div
-    className={css`
-      display: flex;
-      flex-direction: row;
-    `}
-  >
-    {children}
-  </div>
+  <div className={flexRowStyles}>{children}</div>
```

Comment:
We already have Row and Column defined in lots of other places in the codebase and we intend to abstract that into its own component so we should expect to use that here.

Not a blocker, we can define the css outside as well, but when we abstract we should expect this to be replaced